app/Controllers/Account/SecurityController.php

```diff
@@ -122,6 +122,59 @@ public function changePassword()
         ]);
     }
 
+    /**
+     * Enables / disables 2FA (email) for the account.
+     */
+    public function twoFactorAuthEmail()
+    {
+        if (! $this->policy->can('users.twoFactorAuthEmail', auth()->user())) {
+            // user is allowed NOT to change 2FA settings
+            return $this->policy->deny('You are not allowed to change the 2FA settings for this user.');
```

Comment:
It seems it's time to use `alerts()`.

Reply:
Not sure about it. I think it's good as it is. People should see this info only if they manually manipulate the link.